.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+version: 2
+jobs:
+  build-and-test:
+    working_directory: ~/ITKTextureFeatures-build
+    docker:
+      - image: insighttoolkit/module-ci:latest
+    steps:
+      - checkout:
+          path: ~/ITKTextureFeatures
+      - run:
+          name: Configure
+          command: |
+            cmake \
+              -G Ninja \
+              -DITK_DIR:PATH=/ITK-build \
+              -DBUILD_TESTING:BOOL=ON \
+              -DBUILDNAME:STRING=External-ITKTextureFeatures-${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM} \
+                ~/ITKTextureFeatures
+          environment:
```

Comment:
This needs to go in the `Build and Test` section versus the `Configure` section.